llvm/include/llvm/ObjCopy/CommonConfig.h

```diff
@@ -274,6 +274,7 @@ struct CommonConfig {
   bool StripNonAlloc = false;
   bool StripSections = false;
   bool StripUnneeded = false;
+  bool isVerboseEnabled = false;
```

Comment:
This seems to be wrong, doesn't match with the use in ELFObjcopy.cpp

It should just be `bool Verbose = false;`